README.md

```diff
@@ -160,6 +160,43 @@ plex    | s6-supervise avahi: warning: unable to spawn ./run - waiting 10 second
 ```
 As a workaround you can add `- /run` to volumes in your docker-compose.yml or `-v /run` to the docker create command.
 
+## Intel Quick Sync Hardware Transcoding Support
+If your Docker host has access to a supported CPU with the Intel Quick Sync feature set and you are a current Plex Pass subscriber, you can enable hardware transcoding within your Plex Docker container.
+
+A list of current and previous Intel CPU's supporting Quick Sync is available on the Intel [website](https://ark.intel.com/content/www/us/en/ark/search/featurefilter.html?productType=873&0_QuickSyncVideo=True).
+
+Hardware transcoding is a Plex Pass feature that can be added to your Docker container by bind mounting the relevant kernel device to the container. To confirm your host kernel supports the Intel Quick Sync feature, the following command can be executed on the host:
+
+`sudo lspci -v -s $(lspci | grep VGA | cut -d" " -f 1)`
+
+which should output `Kernel driver in use: i915` if Quick Sync is available. To pass the kernel device through to the container, add the device parameter like so:
+
+```
+docker run \
+-d \
+--name plex \
+--network=host \
+-e TZ="<timezone>" \
+-e PLEX_CLAIM="<claimToken>" \
+-v <path/to/plex/database>:/config \
+-v <path/to/transcode/temp>:/transcode \
+-v <path/to/media>:/data \
+--device=/dev/dri:/dev/dri
```

Comment:
Missing the ` \` at the end of the line

Reply:
Resolved in 8dcd1e157b27ae74f88d66e8d4422fdb871852e3